sdk/containerregistry/Azure.Containers.ContainerRegistry/api/Azure.Containers.ContainerRegistry.netstandard2.0.cs

```diff
@@ -198,47 +233,104 @@ public partial class RegistryArtifact
     }
 }
 namespace Azure.Containers.ContainerRegistry.Specialized
 {
+    public abstract partial class ArtifactManifest
+    {
+        protected ArtifactManifest() { }
+        public int? SchemaVersion { get { throw null; } set { } }
+    }
     public partial class ContainerRegistryBlobClient
     {
         protected ContainerRegistryBlobClient() { }
+        public ContainerRegistryBlobClient(string url) { }
+        public ContainerRegistryBlobClient(string url, Azure.Containers.ContainerRegistry.ContainerRegistryClientOptions options) { }
         public ContainerRegistryBlobClient(System.Uri endpoint, Azure.Core.TokenCredential credential, string repository) { }
         public ContainerRegistryBlobClient(System.Uri endpoint, Azure.Core.TokenCredential credential, string repository, Azure.Containers.ContainerRegistry.ContainerRegistryClientOptions options) { }
         public ContainerRegistryBlobClient(System.Uri endpoint, string repository) { }
         public ContainerRegistryBlobClient(System.Uri endpoint, string repository, Azure.Containers.ContainerRegistry.ContainerRegistryClientOptions options) { }
         public virtual System.Uri Endpoint { get { throw null; } }
+        public virtual Azure.Core.Pipeline.HttpPipeline Pipeline { get { throw null; } }
         public virtual string RepositoryName { get { throw null; } }
+        public virtual Azure.Response CancelUpload(string nextLink, Azure.RequestContext context = null) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response> CancelUploadAsync(string nextLink, Azure.RequestContext context = null) { throw null; }
+        public virtual Azure.Response CheckBlobExists(string name, string digest, Azure.RequestContext context = null) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response> CheckBlobExistsAsync(string name, string digest, Azure.RequestContext context = null) { throw null; }
+        public virtual Azure.Response CheckChunkExists(string name, string digest, string range, Azure.RequestContext context = null) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response> CheckChunkExistsAsync(string name, string digest, string range, Azure.RequestContext context = null) { throw null; }
+        public virtual Azure.Response CompleteUpload(string nextLink, string digest, Azure.Core.RequestContent content, Azure.RequestContext context = null) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response> CompleteUploadAsync(string nextLink, string digest, Azure.Core.RequestContent content, Azure.RequestContext context = null) { throw null; }
+        public virtual Azure.Response CreateManifest(string name, string reference, Azure.Core.RequestContent content, string contentType = null, Azure.RequestContext context = null) { throw null; }
```

Comment:
This could be really useful for handling manifests that aren't OciManifests, if needed.  Good idea to keep the protocol method.